services/motion/motion.go

```diff
@@ -188,6 +188,14 @@ type Service interface {
 		slamName resource.Name,
 		extra map[string]interface{},
 	) (bool, error)
+	MoveOnMapNew(
+		ctx context.Context,
+		componentName resource.Name,
+		destination spatialmath.Pose,
+		slamName resource.Name,
+		motionConfig *MotionConfiguration,
```

Comment:
Is adding the motion configuration part of the product spec? If it is, I think nil should be allowed & cause the motion service to use default config.

Reply:
yep, adding a motion configuration is in the scope doc.
I will update such that nil is allowed and causes for the service to use to the default config.